src/components/SplitSection/styled.ts

```diff
@@ -4,6 +4,7 @@ import { usingColors, usingTypography } from 'src/hooks/theme';
 export const Container = styled.div`
   display: flex;
   flex-direction: column;
+  margin-bottom: 20px;
```

Comment:
Assuming that this margin is meant to provide some breathing room at the bottom of the page, then I don't think this is the correct place for it. The app should have some breathing room on all pages, so I'd try moving this to AppWrapper or something instead.

The SplitSection could be used elsewhere in the future, so we should avoid adding any context-specific styling to it.

Reply:
Put bottom padding on the body instead and removed the 20px margin the existing places as well